tests/expectations/alias.compat.c

```diff
@@ -0,0 +1,57 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+enum Status
+#ifdef __cplusplus
+  : uint32_t
+#endif // __cplusplus
+ {
+  Ok,
+  Err,
+};
+#ifndef __cplusplus
+
+typedef uint32_t Status;
+#endif // __cplusplus
+
```

Comment:
Similarly it seems that this newline is a bit excessive.

Reply:
I removed the additional newlines, to make it more compact. It did change one edge case in the `no_includes.cpp` file, that now starts with an empty newline.

```rust
        if !self.functions.is_empty() || !self.globals.is_empty() {
            if self.config.language == Language::C && self.config.cpp_compat {
                out.new_line_if_not_start();
                out.write("#ifdef __cplusplus");
            }

            if self.config.language == Language::Cxx || self.config.cpp_compat {
                out.new_line(); // Used to be out.new_line_if_not_start(), but would cause a double newline when creating C bindings with cpp_compat turned on
                out.write("extern \"C\" {");
                out.new_line();
            }

            if self.config.language == Language::C && self.config.cpp_compat {
                out.write("#endif // __cplusplus");
                out.new_line();
            }
```